test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -195,10 +195,34 @@ public void execute_list_showsAllPersons() throws Exception {
         helper.addToAddressBook(addressBook, false, true);
 
         assertCommandBehavior("list",
-                              Command.getMessageForPersonListShownSummary(expectedList),
-                              expectedAB,
-                              true,
-                              expectedList);
+                Command.getMessageForPersonListShownSummary(expectedList),
+                expectedAB,
+                true,
+                expectedList);
+    }
+
+    @Test
+    public void execute_list_showAllPerson() throws Exception {
```

Comment:
1. title of the test is incorrect. you are testing sort command and not list.
2. the test name doesn't follow the guidelines; please check the naming convention for tests in our coding standards.